pretext/ExtraTopics/Pseudocode.ptx

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-pseudocode">
+  <title>Pseudocode</title>
+
+  <p><term>Pseudocode</term> is a mix of English and code (in our case, Python) that
+    programmers use to plan out their programs.</p>
+
+  <p>Writing pseudocode can help if you want to plan before you write
+    a program, take on a large or confusing programming task, or just as a way to
+    document your code with comments to make it more readable for yourself and others.</p>
+
+  <p>There isn't necessarily a right or wrong pseudocode (except wrong would be if you just wrote the program!).
+    Your pseudocode and language will probably look different from this, but should be along the same lines.</p>
+
+  <p>Take the following prompt for a problem:</p>
+
+  <blockquote>
+    Write a program that takes in a fraction in the format <c>NUMERATOR/DENOMINATOR</c>
+    from the user and divides them <em>and</em> finds their remainder. Print out both results.
```

Comment:
It would be more correct to say they need to output the quotient and the remainder. Those are traditionally the two "results" of division.